### PR TITLE
LLP-25-Contacts

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/pages/contacts.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <h1 className={styles.contactsTitle}>Kontaktai</h1>
+                <div className={styles.contactsPageDescription}>
+                  <p>Sporto g. 18 LT-09238 Vilnius
+                     Telefonas: +370 5 205 5240
+                     El. paštas: info@ltlku.lt
+                  </p>
+                </div>
+              </div>
+              <div className={styles.contactsHeroItem}>
+                <h1 className={styles.contactsTitle}>Įmonės rekvizitai</h1>
```
Comment (pinenetcoder) [inline src/pages/contacts.js:22]: change it to h3 please

File: src/pages/contacts.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+export default function Contacts() {
+   
+   return (
+      <IndexLayout>
+         <main>
+         <section className={styles.contactsPageWrapper}>
+            <div className={styles.contactsPageList}>
+              <div className={styles.contactsHeroItem}>
+                <h1 className={styles.contactsTitle}>Kontaktai</h1>
```
Comment (pinenetcoder) [inline src/pages/contacts.js:13]: change it to h3 please

File: src/pages/contacts.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                     PVM mokėtojo kodas: nėra
+                     SWIFT kodas (BIC): LTKULT21XXX
+                  </p>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section className={styles.contactsCustomerServiceSection}>
+            <div className={styles.contactsCustomerServiceWrapper}>
+               <h2>Klientų aptarnavimas būstinėje</h2>
```
Comment (pinenetcoder) [inline src/pages/contacts.js:35]: change it to h3 please

Comment (mdainys) [reply src/pages/contacts.js]: Done.

**Pull request merged**